cmake/O2Dependencies.cmake

```diff
@@ -639,7 +640,8 @@ o2_define_bucket(
     generators_bucket
 
     DEPENDENCIES
-    Base SimulationDataFormat pythia6 pythia8 MathCore
+    #   Base SimulationDataFormat pythia6 pythia8 MathCore RIO Tree
```

Comment:
no: this is just temporarily. The old setup compiled but had runtime problems. I will create an issue for this to re-enable them.